### DISCUSSION TITLE
Where is docs folder. 😢 There're some typo in wpgraphql.com, And I would like to fix that when i would found.

### DISCUSSION THREAD

**Discussion by jiwon-mun** [wp-graphql/wp-graphql#2700] [General] [answered]
For example, there's a typo in https://www.wpgraphql.com/functions/register_graphql_field. But I can't find docs in this repo.
I assumed it's other repo, but I'm not sure. 

I'm beginner as PHP and WP, so I will have to read this docs, and I would like to give you even a little help.

Please give me a guide to contribute docs area, or where should i fix to do it.

**Comment by justlevine** [comment, marked as answer]
Hey @jiwon-mun !

In general, the docs are located at https://github.com/wp-graphql/wp-graphql/tree/develop/docs. Functions however seem to be a WordPress CPT, which is why you're not finding it there. (Im not sure why its done this way 🤷‍♂️ ).

As such, probably best way to help fix a change would be to open a GH issue (instead of a PR).

**Reply by jiwon-mun** [reply]
Does it mean that there're some docs in another projects? I can't believe this 🥲.
Anyway, thank you so much for kind reply :)

**Reply by justlevine** [reply]
I mean some of the documentation on the website is stored in .MD files in the above link, while others are created manually on wpgraphql.com's WordPress backend.
